packages/admin-next/dashboard/src/v2-routes/store/store-detail/store-detail.tsx

```diff
@@ -0,0 +1,36 @@
+import { Outlet, useLoaderData } from "react-router-dom"
+
+import { JsonViewSection } from "../../../components/common/json-view-section/json-view-section.tsx"
+import { StoreCurrencySection } from "./components/store-currency-section/store-currencies-section.tsx/index.ts"
+import { StoreGeneralSection } from "./components/store-general-section/index.ts"
+import { storeLoader } from "./loader.ts"
+import { useV2Store } from "../../../lib/api-v2/index.ts"
+
+export const StoreDetail = () => {
+  const initialData = useLoaderData() as Awaited<ReturnType<typeof storeLoader>>
+
+  const { store, isLoading, isError, error } = useV2Store({
+    initialData: initialData,
+  })
+
+  if (isLoading) {
+    return <div>Loading...</div>
+  }
+
+  if (isError || !store) {
```

Comment:
Q: There is some places where we use the `usev2Store` but we only check if there is isError but not if store is present, maybe worth a check in the other places above